packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -28,18 +29,36 @@ class DebuggerScreen extends Screen {
 class DebuggerScreenBody extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    if (!kIsWeb) {
+      final theme = Theme.of(context);
+      final textStyle = Theme.of(context)
+          .textTheme
+          .headline
+          .copyWith(color: theme.accentColor);
+      return Center(
+        child: Text(
+          'The debugger screen is only available on web.',
+          style: textStyle,
+        ),
+      );
+    }
+
     // Don't build this in const because compile time const evaluation
     // will fail on non-web apps.
+    // TODO(djshuckerow): Follow up on this: if this code won't compile
+    // on desktop as a const, then the constructor shouldn't be const.
+    // ignore:prefer_const_constructors
```

Comment:
Use conditional imports instead of hacking around the const limitations. 

On the web you would have this Screen widget.
On regular flutter you would have the kIsWeb fallback empty widget.
You can trigger which import is executed based on whether dart:html is available.

Reply:
We should treat this as a bug in Flutter.  Android and iOS native view widgets do not have this issue when building.

I want to make sure that future Flutter for web users don't need to be a special case. 
https://github.com/flutter/flutter/issues/43532

For showing platform views, we should be able to get a pattern like

```dart
switch(platform) {
  case Platform.android:
    return AndroidView(name);
  case Platform.web:
    return HtmlElementView(name);
  case Platform.macos:
    return MacosView(name);
  // etc.
}
```